ruby/.ruby-style.yml

```diff
@@ -556,6 +556,17 @@ Style/PercentLiteralDelimiters:
   Description: 'Use `%`-literal delimiters consistently'
   StyleGuide: 'https://github.com/bbatsov/ruby-style-guide#percent-literal-braces'
   Enabled: true
+  PreferredDelimiters:
+   '%':  ()
+   '%i': ()
+   '%I': ()
+   '%q': ()
+   '%Q': ()
+   '%r': '{}'
```

Comment:
Why do we choose this?

Reply:
And why do the curly braces get wrapped in a string (`'{}'`) when the parentheses don't (`()`)?

Reply:
We need to put the `'` otherwise rubocop is not happy. Must be a YAML thing, these curly braces.
For the "why", I put in the description of the PR why. It is because of regular expressions having often parenthesis.